Client credentials & FS PTLU  toelichting (#680)
Geonovum/KP-APIs: API-strategie-modules/access-control/access-control.md

The [NL GOV Assurance profile for OAuth 2.0](https://gitdocumentatie.logius.nl/publicatie/api/oauth/) supports Client Credentials. The [Forum Standaardisatie Comply-Or-Explain-List](https://www.forumstandaardisatie.nl/open-standaarden/nl-gov-assurance-profile-oauth-20) states when this profile has to be used: 

_The NL GOV Assurance Profile for OAuth 2.0 must be applied to applications where users or resource owners implicitly or explicitly consent to a third-party service to access data via a REST API for which they have access rights._

Client credentials can be used in this specific context for system2system interactions next to Authorization Code Flow, see [NL GOV Assurance profile for OAuth 2.0 - Client Credentials - examples](https://gitdocumentatie.logius.nl/publicatie/api/oauth/v1.1.0-rc.1/#use-case-client-credentials-flow)

> ### Guidance for using NL GOV Assurance Profile for OAuth 2.0 vs Federated Service Connectivity (FCS)

Reviewer: ```suggestion
### Guidance for using NL GOV Assurance Profile for OAuth 2.0 vs Federated Service Connectivity (FSC)
```

Both [NL GOV Assurance Profile for OAuth 2.0](https://gitdocumentatie.logius.nl/publicatie/api/oauth/) and [Federated Service Connectivity (FSC)](https://fsc-standaard.nl/) are relevant standards for access control.
The [Forum Standaardisatie Comply-Or-Explain-List](https://www.forumstandaardisatie.nl/open-standaarden) states when a specific standard must be used. 

> The following guidelines can be used to select the appropriate standard for a specific context:

Reviewer: ```suggestion
The following guidelines should be used to select the appropriate standard for a specific context:
```

1. In the use case a User/Resource owner gives consent to a third-party to access Data
   * YES → NL GOV Assurance Profile for OAuth 2.0
   * NO → go to 2

> 2. In the use case there is data exchange with central systems that are part of the General Digital Infrastructure (GDI) (for example base registries) or the data exchange is cross-sectoral between organizations. Also there is a need for two way authentication.

Reviewer 2: ```suggestion
2. In the use case, data exchange occurs with General Digital Infrastructure (GDI) central systems (e.g., base registries), cross-sectoral between organizations, or requires mutual (two-way) authentication.
```

   * YES → Digikoppeling REST-API profile (using FSC)

>    * NO → Other standards allowed

Reviewer: NO -> Other standards allows (for instance Client Credentials OAuth)

>    * NO → Other standards allowed

Reviewer 3: ```suggestion
   * NO → Other standards allowed (e.g. NLGov OAuth 2.0 client credentials)
```

>    * YES → Digikoppeling REST-API profile with FSC

Reviewer 2: ```suggestion
   * YES → Digikoppeling REST-API profile (using FSC)
```
Anders klinkt FSC als een optie.

>    * NO → (prescribed standard – other choices allowed)

Reviewer 2: ```suggestion
   * NO → Other standards allowed
```

Reviewer: Ik mis nog een "for instance, standard X or Y in the case of A or B"
